klasses/api_test.py

```diff
@@ -187,11 +192,35 @@ def test_fetch_bootcamp_run_dates(end_date_params):
         fetch_bootcamp_run("invalid")
 
 
+@pytest.fixture()
+def patched_create_run_enrollment(mocker):
+    """patched create_run_enrollment"""
+    return mocker.patch("klasses.api.create_run_enrollment", return_value=None)
+
+
+@pytest.fixture()
+def patched_deactivate_run_enrollment(mocker):
+    """patched deactivate_run_enrollment"""
+    return mocker.patch("klasses.api.deactivate_run_enrollment", return_value=[])
+
+
+def test_create_run_enrollments(mocker, patched_create_run_enrollment, user):
+    """test create_run_enrollments"""
+    bootcamp = BootcampFactory.create()
+    bootcamp_runs = BootcampRunFactory.create_batch(3, bootcamp=bootcamp)
+    create_run_enrollments(user, bootcamp_runs)
+    assert patched_create_run_enrollment.call_count == 3
+    expected_calls = []
+    for run in bootcamp_runs:
+        expected_calls.append(mocker.call(user, run, None))
+    patched_create_run_enrollment.assert_has_calls(expected_calls)
+
+
 @pytest.mark.django_db
 @pytest.mark.parametrize("novoed_integration", [True, False])
-def test_create_run_enrollments(mocker, user, settings, novoed_integration):
+def test_create_run_enrollment(mocker, user, settings, novoed_integration):
```

Comment:
This test case should check to see that `can_skip_application_steps` is set to `True`. If you can't guarantee that `can_skip_application_steps` is set to `False` when you're using that `user` fixture, you'll need to use `ProfileFactory` in this test case to create a new user